### PR TITLE
Azure CIS 4.3.5 Ensure server parameter 'connection_throttling' is set to 'ON' for PostgreSQL Database Server

### PR DESCRIPTION
**Pull request by XOmniverse**
Azure CIS 4.3.5 Ensure server parameter 'connection_throttling' is set to 'ON' for PostgreSQL Database Server

Tested and working

### CODE REVIEW

**Review by beagledeltasix — DISMISSED** [review]
Approved.

**Review by beagledeltasix — DISMISSED** [review]
updated global readme looks good.

File: security/azure/pg_conn_throttling/pg_conn_throttling.pt [hunk 1]
```diff
@@ -0,0 +1,193 @@
+name "Azure Ensure PostgreSQL Servers Connection Throttling Enabled"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report if any PostgreSQL server instances do not have connection throttling enabled. \n See the [README](https://github.com/flexera/policy_templates/tree/master/security/azure/pg_conn_throttling) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```
Comment (beagledeltasix) [inline security/azure/pg_conn_throttling/pg_conn_throttling.pt:4]: Shouldn't the doc's link be changed from rightscale to flexera "[docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."

File: security/azure/pg_conn_throttling/pg_conn_throttling.pt [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+###############################################################################
+
+policy "policy_pg_servers_throttling" do
+  validate $ds_bad_pg_servers do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Azure PostgreSQL Server(s) Found Without Connection Throttling Enabled"
+    escalate $esc_email
+    check eq(size(data),0)
+    export do
+      field "name" do
+        label "Name"
```
Comment (beagledeltasix) [inline security/azure/pg_conn_throttling/pg_conn_throttling.pt:166]: Name should be Postgres Instance name or something similar.

File: security/azure/pg_conn_throttling/pg_conn_throttling.pt (hunk 1 above)
Comment (XOmniverse) [reply security/azure/pg_conn_throttling/pg_conn_throttling.pt:4]: It should but I will be doing a mass fix on this once these pull requests are completed.

File: security/azure/pg_conn_throttling/pg_conn_throttling.pt (hunk 2 above)
Comment (XOmniverse) [reply security/azure/pg_conn_throttling/pg_conn_throttling.pt:166]: It's implied that all of the fields are for a Postgres instance by the subject/summary:

Azure PostgreSQL Server(s) Found Without Connection Throttling Enabled

**Review by beagledeltasix — APPROVED** [review]
Approved.

**Review by beagledeltasix — APPROVED** [review]
approved

**Pull request merged**